apps/src/templates/census2017/YourSchool.jsx

```diff
@@ -65,12 +64,8 @@ class YourSchool extends Component {
       schoolForMap = schoolDropdownOption.school;
     }
 
-    // Hide the special announcement.
-    const showSpecialAnnouncement = false;
```

Comment:
I looked this up and it looks like we haven't shown this announcement since 2019 (last blame is on [this commit](https://github.com/code-dot-org/code-dot-org/commit/75d040806625349ed8f22a3d324988a50cff486e)), so this seems safe to remove! 🎉